.gitignore

```diff
@@ -43,3 +43,9 @@ tests/venv
 .cache
 
 docs/generated/
+
+/core/cache-main
```

Comment:
If there are directories under `core/` that need to be ignored, are there not the same directories under other components (e.g. clients, streams, connect, etc)?